assets/sass/patterns/atoms/definition-list.scss

```diff
@@ -17,6 +77,10 @@
   dt {
     @include font-size-and-vertical-height(14, 24);
     @include margin(0, "top");
+
+    &:first-of-type {
```

Comment:
Do we need that truncate-with-ellipsis for the last element, when the text is too long?